web/vtadmin/src/components/routes/Workflows.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2021 The Vitess Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { orderBy } from 'lodash-es';
+import * as React from 'react';
+
+import { useWorkflows } from '../../hooks/api';
+import { useDocumentTitle } from '../../hooks/useDocumentTitle';
+import { DataTable } from '../dataTable/DataTable';
+
+export const Workflows = () => {
+    useDocumentTitle('Workflows');
+    const { data } = useWorkflows();
+
+    const sortedData = React.useMemo(() => orderBy(data, ['cluster.name', 'keyspace.name']), [data]);
+
+    const renderRows = (rows: typeof sortedData) =>
+        rows.map(({ cluster, workflow }, idx) => {
+            return (
+                <tr key={idx}>
+                    <td>{workflow?.name}</td>
+                    <td>{cluster?.name}</td>
+                    <td>{workflow?.source?.keyspace || '-'}</td>
```

Comment:
I wonder if `-` is the right placeholder here, because in vitess that could be confused with the 00-00 shard

Reply:
Ohhh good idea, thanks! I updated it to "n/a" for now, which is straightforward but also lame. I'll get Shannon's thoughts on a better placeholder this week as we go over the rest of the UI.